brownie/network/contract.py

```diff
@@ -127,6 +130,153 @@ def decode_input(self, calldata: Union[str, bytes]) -> Tuple[str, Any]:
 
         return function_sig, input_args
 
+    def get_verification_info(self) -> Dict:
+        """
+        Return a dict with flattened source code for this contract
+        and further information needed for verification
+        """
+        language = self._build["language"]
+        if language == "Vyper":
+            raise TypeError(
+                "Etherscan does not support API verification of source code "
+                "for vyper contracts. You need to verify the source manually"
+            )
+        elif language == "Solidity":
+            flattened_source = ""
+            has_abiencoder = False
+            abiencoder_re = r"(^|\r|\n|\r\n)\s*pragma experimental ABIEncoderV2;"
```

Comment:
Should we move all this parsing logic into a separate class?

Reply:
That is Ben's call :P
The contract.py needs to be refactored anyways at some point... I don't think we should do that in this PR.

Reply:
This probably should only be available on the `ContractContainer`.  Otherwise someone will complain when their contract has a `get_verification_info` function and can't be used with brownie cuz of namespace collision.